### DISCUSSION TITLE
Wemos d1 serial gateway voeden met 3.3 van ducobox?

### DISCUSSION THREAD

**Discussion by Bram81** [arnemauer/Ducobox-ESPEasy-Plugin#108] [General]
Allereerst veel dank voor al je werk en het bouwen van deze plugins! Ik gebruik een wemos d1 mini voor de serial gateway, werkt perfect! Nu vraag ik me af of het niet mogelijk is om de wemos direct te voeden via een 3.3 pin van een van de ducobox sensoren? Het zit nu een beetje onhandig met een micro usb kabel en het zou me weer een adapter schelen. Mijn eigen inschatting is dat die wemos nooit meer trekt dan een van de sensoren van duco zelf, die overigens achterlijk duur zijn.... Of zijn er redenen om dit sterk af te raden?

**Comment by arnemauer** [comment]
Dat zal helaas niet gaan. De voltage regulator die 3.3v maakt geeft in principe maximaal 150mA. 

![powersupply](https://raw.githubusercontent.com/arnemauer/Ducobox-ESPEasy-Plugin/master/Screenshots/Powersupply%20ducobox.jpg)

**Comment by Bram81** [comment]
Hoi sorry voor de late reactie, ik heb blijkbaar geen notificaties aan staan... Ik heb het uiteindelijk gewoon geprobeerd met de voeding van een van de sensoren en de wemos draaide stabiel en zonder problemen.  Na een tijdje heb ik een eigen luchtvochtigheid- en co2-sensor toegevoegd en toen inderdaad wel met een externe voeding gaan werken.  Groeten, Bram
